src/app/page.tsx

```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 export default function Home() {
   return (
```

Comment:
Consider providing a detailed commit message explaining why the `flex` class was removed. This improves code traceability and makes it easier for team members to understand the rationale behind changes.